src/fides/api/alembic/migrations/versions/4fb779cc5f17_add_last_monitored_and_enabled_to_monitor_config.py

```diff
@@ -0,0 +1,36 @@
+"""Add last_monitored and enabled to MonitorConfig
+
+Revision ID: 4fb779cc5f17
+Revises: cb344673f633
+Create Date: 2024-06-17 13:45:46.024496
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "4fb779cc5f17"
+down_revision = "cb344673f633"
```

Comment:
I'm not sure if this `down_revision` is the correct one, since i've rebased my branch a couple of times -- is there a way to manually check this? I've run the migration locally and un-ran it as well and everything seems ok but I was wondering if there's a way to know what the previous migration should be